app/views/layouts/application.html.erb

```diff
@@ -41,16 +41,24 @@
       </a>
 
     <% if user_signed_in? && current_organization.present? %>
-      <%= render partial: "layouts/lte_navbar" %>
+      <% if admin_namespace? %>
+        <%= render partial: "layouts/lte_admin_navbar" %>
+      <% else %>
+        <%= render partial: "layouts/lte_navbar" %>
+      <% end %>
```

Comment:
Ah! I see what you mean -- yeah I agree this is a lot easier to understand, overall. Fewer files too. :100: nice work